Source/KerbalCombatSystems/DecouplerDesignator.cs

```diff
@@ -1,18 +1,52 @@
-﻿namespace KerbalCombatSystems
+﻿using UnityEngine;
+
+namespace KerbalCombatSystems
 {
     public class ModuleDecouplerDesignate : PartModule
     {
-        const string DecouplerDesignationGroupName = "Decoupler Designation";
+        //denote seperator type
+        [KSPField(isPersistant = true)]
+            public string seperatorType = "seperatorType";
+
 
+        [KSPField(isPersistant = true)]
+            public bool seperated = false;
+
+        const string DecouplerDesignationGroupName = "Seperator Designation";
         [KSPField(
             isPersistant = true,
             guiActive = true,
             guiActiveEditor = true,
-            guiName = "Decoupler Type",
+            guiName = "Seperator Type",
             groupName = DecouplerDesignationGroupName,
             groupDisplayName = DecouplerDesignationGroupName),
             UI_ChooseOption(controlEnabled = true, affectSymCounterparts = UI_Scene.None,
-            options = new string[] { "Default", "Weapon", "Countermeasure", "Escape Pod", "Warhead" })]
-        public string DecouplerType = "Default";
+            options = new string[] { "Default", "Escape Pod", "Warhead" })] //re-add "Countermeasure" at later date
```

Comment:
```suggestion
            options = new string[] { "Default", "Warhead", "Escape Pod" })] //re-add "Countermeasure" at later date
```

What do you think of ordering them like this instead?

Reply:
Makes no difference to me